app/src/main/java/cm/aptoide/pt/ApplicationModule.java

```diff
@@ -315,12 +318,16 @@ public ApplicationModule(AptoideApplication application, String aptoideMd5sum) {
       @Named("secureShared") SharedPreferences secureSharedPreferences,
       DownloadsRepository downloadsRepository, InstalledRepository installedRepository,
       @Named("cachePath") String cachePath, @Named("apkPath") String apkPath,
-      @Named("obbPath") String obbPath, DownloadAnalytics downloadAnalytics) {
+      @Named("obbPath") String obbPath, AppInstaller appInstaller,
+      AppInstallerStatusReceiver appInstallerStatusReceiver,
+      InstallerPackageNameManager installerPackageNameManager) {
```

Comment:
I don't know if this name is the best. I understand that it means that it is the installer where we will add the packagename on the installation but i don't know a better name.

Reply:
I agree, can't find a better name than this one: PackageInstallerManager